src/firefly/js/tables/TableUtil.js

```diff
@@ -1403,6 +1403,12 @@ export function getTableState(tbl_id) {
     return TBL_STATE.OK;
 }
 
+export function fixPageSize(pageSize) {
```

Comment:
I think would be good to have a couple of tests for `fixPageSize` to make sure it is doing what expected  (especially providing that we relay on this function in more than one places

Reply:
test added.